defillama/defillama.py

```diff
@@ -330,12 +330,14 @@ def get_fees_chain(self, chain, excludeTotalDataChart=True, excludeTotalDataChar
 
         return self._get(path, params=params)
 
-    def get_fees_protocol(self, protocol, dataType='dailyFees'):
+    def get_fees_protocol(self, protocol, dataType='dailyFees', excludeTotalDataChart=True, excludeTotalDataChartBreakdown=True):
         """
             Get summary of protocol fees and revenue with historical data
         """
-        path = f'/overview/fees/{protocol}'
+        path = f'/summary/fees/{protocol}'
         params = {
+            'excludeTotalDataChart': excludeTotalDataChart,
```

Comment:
this ep doesnt support these params. please remove them.

Reply:
yeah, right, updated